api/services/elastigroup/aws/schemas/elastigroup-deploy-status.yaml

```diff
@@ -34,75 +35,76 @@ properties:
-                batchNum:
-                  type: integer
-                  example: 2
-                status:
-                  type: string
-                  example: DETACHED
-          green:
-            type: array
-            items:
-              type: object
-              properties:
-                instanceId:
-                  type: string
-                  example: i-0b3f1234567890dq3
-                lifeCycle:
-                  type: string
-                  example: spot
-                batchNum:
-                  type: integer
-                  example: 2
-                status:
-                  type: string
-                  example: RUNNING
\ No newline at end of file
+      anyOf:
+        - type: object
+          properties:
+            blue:
+              type: array
+              items:
+                type: object
+                properties:
+                  instanceId:
+                    type: string
+                    example: i-0b3f1234567890ae5
+                  lifeCycle:
+                    type: string
+                    example: spot
+                  batchNum:
+                    type: integer
+                    example: 1
+                  status:
+                    type: string
+                    example: RUNNING
+            green:
+              type: array
+              items:
+                type: object
+                properties:
+                  instanceId:
+                    type: string
+                    example: i-0b3f1234567890dd5
+                  lifeCycle:
+                    type: string
+                    example: spot
+                  batchNum:
+                    type: integer
+                    example: 1
+                  status:
+                    type: string
+                    example: RUNNING
+        - type: object
+          properties:
+            blue:
+              type: array
+              items:
+                type: object
+                properties:
+                  instanceId:
+                    type: string
+                    example: i-0b3f1234567890aa6
+                  lifeCycle:
+                    type: string
+                    example: spot
+                  batchNum:
+                    type: integer
+                    example: 2
+                  status:
+                    type: string
+                    example: DETACHED
+            green:
+              type: array
+              items:
+                type: object
+                properties:
+                  instanceId:
+                    type: string
+                    example: i-0b3f1234567890dq3
+                  lifeCycle:
+                    type: string
+                    example: spot
+                  batchNum:
+                    type: integer
+                    example: 2
+                  status:
+                    type: string
+                    example: RUNNING
```

Comment:
please verify this renders correctly.

Reply:
verified
<img width="407" alt="Screen Shot 2022-08-07 at 13 30 13" src="https://user-images.githubusercontent.com/98812210/183286676-b7de86fe-f2c6-4533-b900-2d6cfda4089d.png">